src/lib/render-lit-html.ts

```diff
@@ -447,9 +456,12 @@ export function* renderTemplateResult(
       case 'custom-element-open': {
         const ctor = op.ctor;
         // Instantiate the element and stream its render() result
-        let instance: HTMLElement | undefined = undefined;
+        let instance = undefined;
         try {
-          instance = new ctor();
+          const element = new ctor();
+          (element as any).tagName = op.tagName;
+          const renderer = new LitElementRenderer(element);
+          instance = {element, renderer};
```

Comment:
At this point, is `instance.element` used for anything?  Seems like if we're abstracting everyting into `renderer`, the "instance" can just be an instance of the renderer?

Reply:
Done.